packages/utilities/src/colors.ts

```diff
@@ -229,21 +229,26 @@ export function updateH(color: IColor, h: number): IColor {
 }
 
 export function updateA(color: IColor, a: number): IColor {
-  return assign({}, color, {
-    a: a,
-    str: (a === 100) ? `#${color.hex}` : `rgba(${color.r}, ${color.g}, ${color.b}, ${a / 100})`
-  });
+  return {
+    ...color, ...(() => {
```

Comment:
how is this function being called? it looks like it just returns the function since there's no () at the end

I think you can just say:
{ ...color, ...{ a: a, str: foobar } }